controllers/htmlRoutes.js

```diff
@@ -249,6 +250,15 @@ router.get("/home", async (req, res) => {
   }
 });
 
+router.post("/delete-account", withAuth, async (req, res) => {
```

Comment:
```suggestion
router.delete("/delete-account", withAuth, async (req, res) => {
```
Should we make this `.delete` instead of `.post`?

Reply:
Also, should we move this method inside the `/api` routes instead? My understanding was that `htmlRoutes` are for rendering html pages only

Reply:
We could, but this is using a default <form> action instead of an async / `fetch()` request. This lets the browser post the form on our behalf without having to write action listeners, button handlers, etc. I am happy to move it to /api/delete-account if we feel that's better! (I will do this now)

 To use a "DELETE" method though we'd need to do it with `fetch()`. That's also okay, if you prefer that; it's just slightly more complicated to achieve the same result

See `action`:  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#attributes_for_form_submission where I read this

Reply:
I moved it to /api/users/deleta-account and updated the forms

Reply:
Oh, good to learn! Thanks for sharing the link!
Since it's more complicated using `fetch`, using `action` seems easier. Thanks for clarifying